django/test/runner.py

```diff
@@ -95,11 +95,11 @@ def check_pickleable(self, test, err):
             pickle.dumps(err)
         except Exception as exc:
             original_exc_txt = repr(err[1])
-            original_exc_txt = textwrap.fill(original_exc_txt, 75)
-            original_exc_txt = textwrap.indent(original_exc_txt, '    ')
+            original_exc_txt = textwrap.fill(
```

Comment:
single line for this would be fine. I adjusted the commit mesage in the PR title. I think it's better to use "Refs" for follow up commits so it doesn't appear as the main fix for the ticket.